x/slashrefund/testslashrefund/testkeeper.go

```diff
@@ -30,7 +30,11 @@ import (
 	tmdb "github.com/tendermint/tm-db"
 )
 
+// NewTestKeeper is used to build all the mocked requirements for testing the slash refund keeper. 
```

Comment:
We should be clearer about mock apps. Since in sdk documentation it is recommended to use mock entities mocked through mockgen, this description may be misleading.